packages/p2p/src/components/app.jsx

```diff
@@ -106,6 +106,7 @@ class App extends Component {
                     residence,
                     agent_id: this.state.agent_id,
                     is_agent: this.state.is_agent,
+                    domain  : this.props.custom_strings.domain || 'deriv.com',
```

Comment:
should we rename it to email_domain so it's clear? since the website domain is deriv.app, not deriv.com

Reply:
thats true, ill make the changes